ceno_zkvm/src/expression.rs

```diff
@@ -26,20 +26,21 @@ use crate::{
 pub enum Expression<E: ExtensionField> {
     /// WitIn(Id)
     WitIn(WitnessId),
-    /// Fixed
+    /// This multi-linear polynomial is known at the setup/keygen phase.
     Fixed(Fixed),
     /// Public Values
     Instance(Instance),
     /// Constant poly
     Constant(E::BaseField),
-    /// This is the sum of two expression
+    /// This is the sum of two expressions
     Sum(Box<Expression<E>>, Box<Expression<E>>),
-    /// This is the product of two polynomials
+    /// This is the product of two expressions
     Product(Box<Expression<E>>, Box<Expression<E>>),
-    /// This is x, a, b expr to represent ax + b polynomial
-    /// and x is one of wit / fixed / instance, a and b are either constant or challenge
+    /// ScaledSum(x, a, b) to represent a * x + b
```

Comment:
Why not just
ScaledSum(x, a, b) **represents** a * x + b?

Reply:
Sure, that sounds better. I was just preserving the original wording here.